src/tlo/methods/healthsystem.py

```diff
@@ -2543,13 +2635,16 @@ def process_events_mode_2(self, hold_over: List[HSIEventQueueItem]) -> None:
                             self.module.running_total_footprint += updated_call
 
                             # Write to the log
+                            # WARNING: the logged appt footprint does not contain information
+                            # on whether task-shifting was performed or not.
```

Comment:
indeed, this would be another reason to let the HSI return actual footprint that its provided with

Reply:
See reply to comment above